iml-manager-cli/src/nginx.rs

```diff
@@ -108,6 +108,7 @@ mod tests {
             ("GRAFANA_PROXY_PASS", "http://127.0.0.1:3000"),
             ("INFLUXDB_PROXY_PASS", "http://127.0.0.1:8086"),
             ("TIMER_PROXY_PASS", "http://127.0.0.1:8892"),
+            ("BRANDING_PATH", "ddn-favicons"),
```

Comment:
Can we change this to ddn? Eventually we should move the logos out of the binary as well.